### ISSUE TITLE
Fortran Lexer Out of Date

### ISSUE THREAD

**Issue by smf007** [alecthomas/chroma#645]
**What problem does this feature solve?**

There have been a couple updates to the pygments lexer for fortran that are not reflected in chroma's versions.
For example, https://github.com/pygments/pygments/pull/1877

**What feature do you propose?**

Running pygments2chroma.py generates an updated lexer, however the resulting file is different than https://github.com/alecthomas/chroma/blob/master/lexers/embedded/fortran.xml so I'm not sure how to get the correct format in order to apply the change and make a pull request.

**Comment by alecthomas** [comment]
Those changes are small enough that they could just be manually applied to the XML lexer I think.

**Comment by smf007** [comment]
Looking at it more closely, there are a number of additional changes beyond that pull request I noted above.

If there is no automated processing to create the xml file, I'll try to do that by hand.  README.md section on lexar could use an explanation about the relationship between the .go version and the .xml embedded versions.  I'm just assuming the .xml takes precedence over any .go file.

**Comment by smf007** [comment]
Tried to do the update.  Appears to work.  Pull #646

**Comment by alecthomas** [comment]
Yeah I have been meaning to update the importer to output xml but it is quite a bit of work

**Comment by smf007** [comment]
I just want to make sure I understand the "state" when using the xml.

So the tuple would be
> {`\.(true|false)\.`,`` NameBuiltin, nil}

In the xml version, would the "nil" state be the default?  i.e. not specified is "nil"
